setup.py

```diff
@@ -49,17 +50,10 @@
     'SQLAlchemy>=1.1.9',
     'pygraph>=0.2.1',
     'unicodecsv>=0.14.1',
-]
-
-SETUP_REQUIRES = [
+    # Setup.
     'google-apputils>=0.4.2',
     'python-gflags>=3.1.1',
-    'grpcio',
```

Comment:
These should also be in setup.py. If already installed using the script they won't be recompiled from source. But someone just coming and installing from setup.py will get all required dependencies.

Reply:
Done and confirmed they're not rebuilt if you're on Travis/Docker.